src/daft-physical-plan/src/physical_planner/translate.rs

```diff
@@ -938,6 +939,92 @@ pub fn populate_aggregation_stages(
+                );
+                let sum_id = add_to_stage(AggExpr::Sum, se.clone(), schema, &mut first_stage_aggs);
+                let squared_sum_id = add_to_stage(
+                    |se| AggExpr::Sum(se.clone().mul(se)),
+                    se.clone(),
+                    schema,
+                    &mut first_stage_aggs,
+                );
+                let cubed_sum_id = add_to_stage(
+                    |se| AggExpr::Sum(se.clone().mul(se.clone()).mul(se)),
+                    se.clone(),
+                    schema,
+                    &mut first_stage_aggs,
+                );
+
+                // Second stage: sum(count), sum(sum), sum(squared_sum), sum(cubed_sum)
+                let sum_count_id = add_to_stage(
+                    AggExpr::Sum,
+                    resolved_col(count_id),
+                    schema,
+                    &mut second_stage_aggs,
+                );
+                let sum_sum_id = add_to_stage(
+                    AggExpr::Sum,
+                    resolved_col(sum_id),
+                    schema,
+                    &mut second_stage_aggs,
+                );
+                let sum_squared_sum_id = add_to_stage(
+                    AggExpr::Sum,
+                    resolved_col(squared_sum_id),
+                    schema,
+                    &mut second_stage_aggs,
+                );
+                let sum_cubed_sum_id = add_to_stage(
+                    AggExpr::Sum,
+                    resolved_col(cubed_sum_id),
+                    schema,
+                    &mut second_stage_aggs,
+                );
+
+                // Final projection: Given
+                // - sum(count) = N
+                // - sum(sum) = S
+                // - sum(squared_sum) = S2
+                // - sum(cubed_sum) = S3
+                //         S3 - 3 * S2 * S / N + 2 * S^3 / N^2
+                // Skew = -------------------------------------
+                //          N * ((S2 - S^2 / N) / N) ^ (3/2)
+
+                let n = resolved_col(sum_count_id);
+                let s = resolved_col(sum_sum_id);
+                let s2 = resolved_col(sum_squared_sum_id);
+                let s3 = resolved_col(sum_cubed_sum_id);
+
+                let denom_base = s2
+                    .clone()
+                    .sub(s.clone().mul(s.clone()).div(n.clone()))
+                    .div(n.clone());
+                let denom = sqrt::sqrt(denom_base.clone().mul(denom_base.clone()).mul(denom_base));
+
+                let numerator = s3.sub(lit(3).mul(s2).mul(s.clone()).div(n.clone())).add(
+                    lit(2)
+                        .mul(s.clone())
+                        .mul(s.clone())
+                        .mul(s)
+                        .div(n.clone())
+                        .div(n.clone()),
+                );
+
+                let result = numerator.div(denom).div(n).alias(output_name);
+                final_exprs.push(result);
```

Comment:
I did it like this using the DuckDB algorithm instead of just accumulation and calling skew because I couldn't really figure out how to implement a `list_skew` on ListArrays. Kept running into issues

Reply:
If you happen to think that would be easier or have guidance for the next time, just let me know